### PR TITLE
[WIP] Check all importance types for lightgbm

### PR DESCRIPTION
**Pull request by lopuhin**
Trying to write a test for #215 - want to check if this passes on Travis too.

### CODE REVIEW

**Comment by lopuhin** [comment]
new tests pass, but old explain_prediction tests fail (https://travis-ci.org/TeamHG-Memex/eli5/jobs/243194617#L785) in the same way as locally, likely due to some recent LightGBM changes

**Comment by kmike** [comment]
Merged as a part of https://github.com/TeamHG-Memex/eli5/pull/222. Thanks @lopuhin!

**Comment by lopuhin** [comment]
Thanks for fixing all the issues @kmike !
